src/sentry/api/endpoints/user_authenticator_enroll.py

```diff
@@ -150,6 +152,16 @@ def post(self, request, user, interface_id):
 
         :auth: required
         """
+        if ratelimiter.is_limited(
+            u"auth:authenticator-enroll:{}".format(request.user.id),
```

Comment:
If we could somehow include the identified device (phone number, for SMS's example), I'd say we should tie that into this limiter and crank it up to once per user+device per hour.

Reply:
After some deliberation, let's use the authentication mechanism in here instead of my previous suggestions (sms vs otp vs u2f) and crank this badboy up to 10 per type per day.

Reply:
Sg2m